test/Swashbuckle.AspNetCore.SwaggerGen.Test/Fixtures/FakeGenericResourceController.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Threading;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Swashbuckle.AspNetCore.SwaggerGen.Test.Fixtures
```

Comment:
For new files you've added, could use use file-scoped namespaces?

Reply:
@martincostello, we could apply an editorConfig setting in order to get this automatically:
csharp_style_namespace_declarations = file_scoped:suggestion